# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/debug_render.rs
# Commit: 5589923a52c8
# Full Hash: 5589923a52c80ed09c8ce383afb0385cabc0ec18
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-15 09:29:54
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 2 - Remove texture array usage from render task graph. r=nical
#   
#   With this change, all color/alpha intermediate surfaces are individual
#   textures, rather than texture arrays.
#   
# ==============================================================================

diff -r 2d192d43277b -r 5589923a52c8 gfx/wr/webrender/src/debug_render.rs
--- a/gfx/wr/webrender/src/debug_render.rs	Tue Dec 15 00:01:19 2020 +0000
+++ b/gfx/wr/webrender/src/debug_render.rs	Tue Dec 15 00:43:28 2020 +0000
@@ -140,7 +140,7 @@
         let tri_vao = device.create_vao(&DESC_COLOR);
 
         let font_texture = device.create_texture(
-            ImageBufferKind::Texture2DArray,
+            ImageBufferKind::Texture2D,
             ImageFormat::R8,
             debug_font_data::BMP_WIDTH,
             debug_font_data::BMP_HEIGHT,